website/blog/2023-12-05-ariadne-codegen-0-11.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ariadne Codegen 0.11
+---
+
+Ariadne Codegen 0.11 is now available!
+
+This release brings improvements and new options in the areas of performance and default base clients.
+
+
+<!--truncate-->
+
+## Removed `model_rebuild` calls
+
+Pydantic v2 has changed the way forward refs are handled during model class initialisation. If an annotation couldn't be resolved, an auto-rebuild is performed during the first validation attempt. Based on this, in this release we're removing all `model_rebuild` calls from the generated package.
+
+
+## Generating only used inputs and enums
+
+Version 0.11 introduces 2 new configuration flags that can be used to reduce the number of unused generated models:
+
+- `include_all_inputs` (defaults to `true`) - when set to `true` the generated package will include a model for every input defined in the schema, otherwise it will only create inputs used as arguments in provided operations.
```

Comment:
```suggestion
- `include_all_inputs` (defaults to `true`) - when set to `true` the generated package will include a model for every input defined in the schema, otherwise it will only generate inputs used by defined operations.
```